webpack.common.js

```diff
@@ -45,16 +47,22 @@ var config = {
       {
         test: /\.tsx?$/,
         exclude: [/node_modules/],
-        loaders: ['ts-loader']
+        use: ["ts-loader"]
       },
       {
         test: /\.(png|woff|woff2|eot|ttf|svg).*$/,
-        loader: 'url-loader?limit=100000'
+        use: ["url-loader?limit=100000"]
       }
     ],
   },
   resolve: {
     extensions: [".ts", ".tsx", ".js", ".scss"],
+    fallback: {
+      buffer: require.resolve("buffer"),
```

Comment:
Is the `buffer` polyfill needed for the production deployment? It is currently listed as a dev dependency.

Reply:
🙏🏽 Oh, and I meant to mention that I really appreciate the explanatory call-out comments. They make it really easy to reason over changes and substantially reduce my research time and cognitive load!

Reply:
It isn't needed for production, but this should be fine. `resolve.fallback.buffer` doesn't necessarily cause `buffer` to be included. It says that if something imports `buffer`, but it can't be found (in this case, because it's expected to be available by default in a node environment), then resolve it to the `buffer` from our package.json. In a production build, nothing imports `buffer`, so it wouldn't be included.